internal/settings/manager.go

```diff
@@ -0,0 +1,35 @@
+package settings
+
+import (
+	"context"
+	"fmt"
+
+	promoterv1alpha1 "github.com/argoproj-labs/gitops-promoter/api/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type ManagerConfig struct {
+	GlobalNamespace                  string
+	GlobalPromotionConfigurationName string
```

Comment:
Maybe we should just hard-code a constant? I can't think of a reason someone would need to change this.

Reply:
I want to in the future add instance-id support (for many reasons) which means you might be running multiple controllers in one namespace.

Reply:
Constant for now, configurable when we add instance ID? Seems better to code it all at once.

Reply:
That's fair

Reply:
Moved it to a constant